packages/widget/src/modals/AssetAndChainSelectorModal/AssetAndChainSelectorModalSearchInput.tsx

```diff
@@ -65,6 +69,7 @@ export const AssetAndChainSelectorModalSearchInput = ({
 
       <StyledSearchInput
         ref={inputRef}
+        autoFocus
```

Comment:
I really don't know why but adding this attribute seems necessary to not autoFocus on mobile